force_bdss/core_plugins/dummy/kpi_adder/kpi_adder_calculator.py

```diff
@@ -16,3 +17,15 @@ def run(self, model, data_source_results):
                 type=model.cuba_type_out,
                 value=sum
             )]
+
+    def slots(self, model):
+        return (
+            (
+                Slot(type=model.cuba_type_in),
```

Comment:
We could already have a dynamic number of slots, depending on the model ? For the KPIAdder we could have a `model.number_inputs` integer